edvart/report_sections/dataset_overview.py

```diff
@@ -202,10 +192,15 @@ def add_cells(self, cells: List[Dict[str, Any]]) -> None:
 
         if self.verbosity == Verbosity.LOW:
             code = "overview_analysis(df=df"
-            if self.subsections_low_verbosity is not None:
+            subsections_to_show_with_low_verbo = [
+                sub
+                for sub in self.subsections_to_show
+                if self.subsection_verbosities[sub] == Verbosity.LOW
+            ]
+            if subsections_to_show_with_low_verbo != self._DEFAULT_SUBSECTIONS_TO_SHOW:
                 arg_subsections_names = [
                     f"Overview.OverviewSubsection.{str(sub)}"
-                    for sub in self.subsections_low_verbosity
+                    for sub in subsections_to_show_with_low_verbo
```

Comment:
Nitpick/OCD: The word `verbo` instead of full `verbosity`. I would use the full word if it doesn't break some formatting due to being too long.

Reply:
Changed